README.md

```diff
@@ -1,12 +1,9 @@
 # LazyBroadcast.jl
 
-We believe that software should be
-
- - simple
- - unit tested
- - fast
- - granular
+This package helps users write faster and more unit-test-friendly code, by
+avoiding temporary allocations, fusing operations, and enabling a more
+functional-programming design with Julia's broadcasting.
```

Comment:
I would say that:

`LazyBroadcast.jl` provides tools to defer the execution of Julia broadcasted expressions. This can be used to avoid temporary allocations, fuse operations, and enable a even more functional-programming approaches to Julia packages.

Reply:
Sounds good to me